src/query/graphite/lexer/lexer_test.go

```diff
@@ -0,0 +1,142 @@
+package lexer
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type lexerTestData struct {
+	input          string
+	expectedTokens []Token
+	reservedIdents map[string]TokenType
+}
+
+func TestLexer(t *testing.T) {
```

Comment:
Similar for this method, need to remove all of the non-test data IDs and make them more synthetic/test data like (e.g. foo/bar/baz/etc)